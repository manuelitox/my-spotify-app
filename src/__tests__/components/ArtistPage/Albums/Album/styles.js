import React from 'react'
import { shallow } from 'enzyme'

import StylesAlbum from 'components/ArtistPage/Albums/Album/styles'

describe('Component | StylesAlbum', () => {
  let component 
  beforeEach( () => {
    component = shallow(
      <StylesAlbum />
    )
  })
  it('should render a styled component', () => {
    expect(component).toMatchSnapshot()
  })
})