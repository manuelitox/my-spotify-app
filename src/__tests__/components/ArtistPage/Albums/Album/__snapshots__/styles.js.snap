// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | StylesAlbum should render a styled component 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  cursor: pointer;
  display: grid;
  grid-template-columns: 60px 2fr;
  grid-gap: 20px;
  img {
    border-radius: 4px;
  }
  h3 {
    color: white;
    font-size: 1.6rem;
    font-weight: 500;
    margin: 0 0 10px;
  }
  p {
    color: ",
          [Function],
          ";
    font-size: 1.4rem;
    margin: 2px 0;
  }
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
